src/main/kotlin/com/cosmotech/api/rbac/CsmRbac.kt

```diff
@@ -114,8 +128,17 @@ open class CsmRbac(
   }
 
   fun getAccessControl(rbacSecurity: RbacSecurity, userId: String): RbacAccessControl {
+    return getAccessControlOrThrow(
+        rbacSecurity, userId, "User '$userId' not found in ${rbacSecurity.id} component")
+  }
+
+  fun getAccessControlOrThrow(
+      rbacSecurity: RbacSecurity,
+      userId: String,
+      exceptionUserNotFoundMessage: String
+  ): RbacAccessControl {
     return rbacSecurity.accessControlList.find { it.id == userId }
-        ?: throw CsmResourceNotFoundException("User '$userId' not found")
+        ?: throw CsmResourceNotFoundException(exceptionUserNotFoundMessage)
   }
```

Comment:
I'm not sure that you need two methods here. The error message `"User '$userId' not found in ${rbacSecurity.id} component"` is sufficient in my opinion as `rbacSecurity.id` refers to a formatted id (0-XXXXXX for organisation, W-XXXXX for workspace,...)
I think you can keep the method as it was designed and just change the error message without created a new one, i.e:
```
fun getAccessControl(rbacSecurity: RbacSecurity, userId: String): RbacAccessControl {
return rbacSecurity.accessControlList.find { it.id == userId }
        ?: throw CsmResourceNotFoundException("User '$userId' not found in ${rbacSecurity.id} component")
  }
  ```